cmd/zoekt-sourcegraph-indexserver/debug.go

```diff
@@ -102,8 +102,9 @@ func debugMerge() *ffcli.Command {
 
 func debugList() *ffcli.Command {
 	fs := flag.NewFlagSet("debug list", flag.ExitOnError)
-	conf := rootConfig{}
-	conf.registerRootFlags(fs)
+
+	hostname := fs.String("hostname", "localhost", "the hostname of the zoekt-sourcegraph-indexserver instance to connect to")
+	listen := fs.String("listen", ":6072", "indexserver is listening on this port")
```

Comment:
> I'm not sure of the benefit of this change. If I am directly calling zoekt-sourcegraph-indexserver, it might as well directly run the code instead of running the same code behind an RPC? This now requires a running indexserver, while previously all we needed was pointing at a valid sourcegraph URL or a valid IndexDir.
> 
> Is there a benefit I am missing?

Not sure I understand entirely. If the question is whether we can we remove those debug commands because they don't add much value over directly calling curl, then yes! Shall we remove them?

Reply:
I think the value they provide is discoverability once you are on the pod. Granted, I don't expect many admins to be on the pod, only us. And in that case we are power users enough to directly run curl. So lets just remove! :)